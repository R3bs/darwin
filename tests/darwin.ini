# Darwin Submit file example
#   
#   This file represents the way any user will submit any given file to the darwin 
#   application. The executables should inform the absolute path of all files
#
#
#


[darwin]
Executable=/bin/sleep

[htcondor]

Collector=lsc.ic.unicamp.br
Universe=vanilla
Error=logs/err.$(cluster)
Output=logs/out.$(cluster)
Log=logs/log.$(cluster)

initialdir=files

should_transfer_files = YES
when_to_transfer_output = ON_EXIT
#transfer_input_files = files/in1,files/in2

#Arguments       = in1 in2 out1

Requirements=( TARGET.Arch == "X86_64" ) && ( TARGET.OpSys == "LINUX" ) && ( TARGET.Disk >= 0 ) && ( TARGET.Memory >= RequestMemory ) && ( ( TARGET.HasFileTransfer ) || ( TARGET.FileSystemDomain == MY.FileSystemDomain ) )
Rank=memory
Image_Size=28000

