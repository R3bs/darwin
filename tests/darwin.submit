# Darwin Submit file example
#   
#   This file represents the way any user will submit any given file to the darwin 
#   application. The executables should inform the absolute path of all files
#

[darwin]

# define the collector from htcondor to use
collector=lsc.ic.unicamp.br

# always refresh on 'x' seconds, default: 60
refresh_rate=60

# define the job file structure to be created
dirs=logs

# define the strategy to monitor agent job
# can be: pool; event
agent_monitoring=pool

# here we use all methods to delegate to condor
# although the user shoukld take care of defining the directoires
# and how the program work
[htcondor]

executable=/bin/sleep

universe=vanilla

error=%(darwin_ag)s/logs/err.$(ClusterId)

output=%(darwin_ag)s/logs/out.$(ClusterId)

log=%(darwin_ag)s/logs/log.$(ClusterId)

initialdir=files

# in the case where the filesystem is not shared, if shared use NO
should_transfer_files = YES

when_to_transfer_output = ON_EXIT
#transfer_input_files = files/in1,files/in2
#transfer_output_files = %(darwin_ag)s/files/

#Arguments       = in1 in2 out1

requirements=( TARGET.Arch == "X86_64" ) && ( TARGET.OpSys == "LINUX" ) && ( TARGET.Disk >= 0 ) && ( TARGET.Memory >= RequestMemory ) && ( ( TARGET.HasFileTransfer ) || ( TARGET.FileSystemDomain == MY.FileSystemDomain ) )
rank=memory
image_Size=28000

