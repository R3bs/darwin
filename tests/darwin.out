sleeping for 30 seconds
sleeping for 30 seconds
